README.md

```diff
@@ -13,23 +13,35 @@ Install `eslint` and `eslint-config-exp`:
 npm install --save-dev eslint eslint-config-exp
 ```
 
-Add the following to your `.eslintrc.json`-file:
+For CommonJS modules, add the following to your `.eslintrc.json`-file:
 
 ```json
 {
   "root": true,
-  "extends": ["eslint-config-exp"]
+  "extends": [ "eslint-config-exp" ]
+}
+```
+
+If you are using ES6 modules, instead add this to the `.eslintrc.json`-file:
+
+```json
+{
+  "root": true,
+  "extends": [ "eslint-config-exp/modules" ]
```

Comment:
I would rather see this be extended with multiple sets instead of having one separate like the following:
```suggestion
  "extends": [ "eslint-config-exp", "eslint-config-exp/modules" ]
```

Reply:
Okay! Both would work, I think I am leaning the other way. Maybe a tie-breaker comment from someone? 😄

Reply:
I saw that Facebooks react config uses this type of extending for their test config: https://github.com/facebook/create-react-app/tree/main/packages/eslint-config-react-app
Would it in that case be good to separate them all into `base`, `modules`, `test` - and users can import any they want?

Reply:
In my head I'm imagining that the base rules set should be CommonJS support in a node environment. If you import the extension for ES modules, then the parser options should reflect that by overriding it; as well as any other rules that a) are no longer needed or b) are now required.

The whole point about this module is to provide a consistent way of linting our projects and so it seems to me that the modules extension should be an add-on.

Reply:
If I'm not misunderstanding you, the end result is the same. Either:

- the two configs define everything, and we use one of them
- one config extends the next, and we use both of them in the case we want module support

I pushed changes to do it the way you suggested, since I saw that this was the approach of create-react-app.

Consistent linting will be done no matter which of the two approaches we choose though.